staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -514,6 +516,19 @@ func convertv1alpha1Validations(inputValidations []v1alpha1.Validation) []cel.Ex
 	return celExpressionAccessor
 }
 
+func convertV1Alpha1MessageExpressions(inputValidations []v1alpha1.Validation) []cel.ExpressionAccessor {
+	celExpressionAccessor := make([]cel.ExpressionAccessor, len(inputValidations))
+	for i, validation := range inputValidations {
+		if validation.MessageExpression != "" {
```

Comment:
anywhere this is "" we're leaving nil entries in the returned celExpressionAccessor list... that's really subtle and could easily lead to NPE bugs

Reply:
This is a compromise of a refactor that other ongoing development relies on (see reply above). This is the least intrusive way to do placeholders. I will send another refactor once affected developments finish.